src/main/java/org/mondemand/Client.java

```diff
@@ -71,6 +69,7 @@
   private ConcurrentHashMap<EventType, List<Transport>> transports = null;
   private ClientStatEmitter autoStatEmitter = null;
   private Thread emitterThread = null;
+  private Integer maxNumMetrics = null;
```

Comment:
Well, it may be a little bit weird it being here at all, actually.  This will get used in addTransportsFromConfigFile when it constructs LWESTransports from that.  There may be users of this library that are instantiating their own LWESTransports directly though, so I though a default in the transport made sense.